ersilia/cli/commands/catalog.py

```diff
@@ -43,15 +45,25 @@ def catalog(local=False, file_name=None, browser=False, more=False):
             mc.airtable()
             return
         if local:
+            catalog_table = mc.local()
```

Comment:
I like your use of storing `mc.local` and `mc.hub` into a `catalog_table` variable. Could we maybe rewrite this whole bit as:

```Python

catalog_table = mc.local() if local else mc.hub()

if file_name is None:
    catalog = catalog_table.as_json()
else:
    catalog_table.write(file_name)
    catalog = None
```

Reply:
Alright